apps/src/lib/ui/accounts/ChangeEmailForm.jsx

```diff
@@ -90,30 +106,33 @@ export default class ChangeEmailForm extends React.Component {
             ref={el => this.currentPasswordInput = el}
           />
         </Field>
-        <Field
-          label={i18n.changeEmailModal_emailOptIn_description()}
-          error={validationErrors.emailOptIn}
-          style={{display: 'none'}}
-        >
-          <select
-            value={values.emailOptIn}
-            onKeyDown={this.onKeyDown}
-            onChange={this.onEmailOptInChange}
-            disabled={disabled}
-            style={{
-              ...styles.input,
-              width: 100,
-            }}
+        {userType === 'teacher' &&
+          <Field
+            labelDetails={this.emailOptInLabelDetails()}
+            error={validationErrors.emailOptIn}
           >
-            <option value=""/>
-            <option value="yes">
-              {i18n.yes()}
-            </option>
-            <option value="no">
-              {i18n.no()}
-            </option>
-          </select>
-        </Field>
+            <select
+              value={values.emailOptIn}
+              disabled={disabled}
+              tabIndex="1"
+              onKeyDown={this.onKeyDown}
+              onChange={this.onEmailOptInChange}
+              style={{
+                ...styles.input,
+                width: 100,
+              }}
+              ref={el => this.emailOptInSelect = el}
```

Comment:
I see that `el` is common usage, but what does it stand for?

Reply:
Abbr. for `DOMElement`.  You can actually see this exact abbreviation used in [React's own documentation for callback refs.](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs)

Fun fact: As of React 16.3 callback refs are no longer the preferred way to create refs - instead you're supposed to use the [`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs) API.  We're not up to that version yet, though.